app/src/main/java/com/github/kiolk/cowsandbulls/App.java

```diff
@@ -46,6 +48,15 @@ public static SettingsRepository getSettingsRepository() {
         return settingRepository;
     }
 
+    public static Prefs getPrefsInstance() {
```

Comment:
Please see how work settings repository. Use Setting repository for store setting of application. Remove this code.